src/osgearth-1-fixes.patch

```diff
@@ -0,0 +1,59 @@
+This file is part of MXE. See LICENSE.md for licensing information.
+
+Contains ad hoc patches for cross building.
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Boris Pek <tehnick-8@yandex.ru>
+Date: Fri, 16 Mar 2018 17:09:37 +0300
+Subject: [PATCH 1/2] disable usage of extra libraries
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 1111111..2222222 100755
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -117,10 +117,10 @@ ENDIF (OSGEARTH_USE_GLES)
+ 
+ FIND_PACKAGE(CURL)
+ FIND_PACKAGE(GDAL)
+-FIND_PACKAGE(GEOS)
++#FIND_PACKAGE(GEOS)
+ FIND_PACKAGE(Sqlite3)
+ FIND_PACKAGE(ZLIB)
+-FIND_PACKAGE(Poco)
++#FIND_PACKAGE(Poco)
```

Comment:
Can these be enabled? It's preferable to enable as many features as possible.

If not, can they be made options and sent upstream? It will make future updates easier (also for apps and tests below)

Reply:
> Can these be enabled? It's preferable to enable as many features as possible.

Yes, sure. I just forgot to drop this part of patch during preparing of PR. (I do not use this part of osgearth in my MXE fork.)

> If not, can they be made options and sent upstream? It will make future updates easier (also for apps and tests below)

Ok. I'll do this for apps and tests.